inc/classes/class-imagify-admin-ajax-post.php

```diff
@@ -363,21 +363,21 @@ public function imagify_optimize_missing_sizes_callback() {
 	 *
 	 * @since 1.9
 	 */
-	public function imagify_generate_webp_versions_callback() {
+	public function imagify_generate_nextgen_versions_callback() {
```

Comment:
:warning: This function has been refactored here: 
https://github.com/wp-media/imagify-plugin/blob/ffcbe66cfa06498264a8897d398357a7a9b97d7d/inc/classes/class-imagify-admin-ajax-post.php#L366